prepare-videochat.sh

```diff
@@ -359,7 +359,7 @@ if ! has_kernel_module v4l2loopback; then
 fi
 
 # Probe module if not probed yet
-if lsmod | grep v4l2loopback >/dev/null 2>/dev/null; then
+if lsmod | grep -w v4l2loopback >/dev/null 2>/dev/null; then
```

Comment:
While we're at it, could you see if we can replace this with `if has_kernel_module v4l2loopback; then`?